python/semantic_kernel/planning/plan.py

```diff
@@ -26,10 +26,10 @@
 logger: logging.Logger = logging.getLogger(__name__)
 
 
-class Plan(KernelFunctionBase):
+class Plan:
```

Comment:
will this not break type checking? when a plan get's invoked in the place of a function? so this can then inherit from KernelFunction?

Reply:
We spoke offline about this. Type checking isn't breaking related to removing this inheritance.